pkg/volume/flexvolume/flexvolume.go

```diff
@@ -98,6 +98,10 @@ func (plugin *flexVolumePlugin) RequiresRemount() bool {
 	return false
 }
 
+func (plugin *flexVolumePlugin) SupportsMountOption() bool {
+	return false
```

Comment:
Since the mount is expected to be handled directly by the FLEX script, i dont think we want to take mount options in the FLEX volume plugin just yet.  We'd have to pass them through to the FLEX shell script mount command.  An easier solution if the storage admin/storage vendor wanted mount options they could edit their FLEX script directly to include them.